oauth2/handler.go

```diff
@@ -125,10 +125,19 @@ func OnLogin(c LoginCallback) Param {
 	}
 }
 
+// SetRedirectURI sets the redirect URL.
+func SetRedirectURI(uri string) Param {
```

Comment:
```suggestion
// WithRedirectURL sets  a static redirect URL. By default, the redirect URL is
// generated using the request path, the Host header, and the ForceTLS option.
func WithRedirectURL(uri string) Param {
```

This will match the other `Param` names better and also clarify that this is option not required.

Reply:
Makes sense, thanks